FormlyMapper/FormlyField.php

```diff
@@ -35,9 +35,15 @@ public function getFormlyFieldConfiguration()
      */
     protected function buildConfiguration()
     {
+        $this->formlyFieldConfiguration = [];
```

Comment:
But you never retrieve a `FormlyField` directly since it's an abstract class, so that shouldn't be.. Each field should be a new instance of a concrete class that extends `FormlyField`, and thus would have it's own `$this->formlyFieldConfiguration` defaulted to `[]` already. Unless I'm missing something here.